pkg/kobject/kobject.go

```diff
@@ -99,8 +98,20 @@ type ServiceConfig struct {
 	Dockerfile      string              `compose:"dockerfile"`
 	Replicas        int                 `compose:"replicas"`
 	GroupAdd        []int64             `compose:"group_add"`
-	// Volumes is a struct which contains all information about each volume
-	Volumes []Volumes `compose:""`
+	Volumes         []Volumes           `compose:""`
+	HealthChecks    HealthCheck         `compose:""`
+}
+
+// HealthCheck the healthcheck configuration for a service
+// "StartPeriod" is not yet added to compose, see:
+// https://github.com/docker/cli/issues/116
+type HealthCheck struct {
+	Test        []string
+	Timeout     int32
+	Interval    int32
+	Retries     int32
+	StartPeriod int32
```

Comment:
Umm, should we add StartPeriod since it's not a part of upstream Compose, yet? I understand that soon it will be, but today this is invalid Compose syntax.
Should we just comment it out for now and enable it as soon as it's merged?
Also, an issue to track this would help.